README.md

```diff
@@ -1,9 +1,5 @@
 # USDR: Grants One Stop Tool (GOST)
 
-[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](./CODE_OF_CONDUCT.md)
-[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](./LICENSE)
-[![Build Status](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/ci.yaml/badge.svg)](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/ci.yaml)
```

Comment:
Alternative, if we really like badges and want to keep them...

This markdown:
```markdown
# USDR: Grants One Stop Tool (GOST)

[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](./CODE_OF_CONDUCT.md)
[![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](./LICENSE)
[![Build Status](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-staging.yml/badge.svg)](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-staging.yml)
[![Build Status](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-production.yml/badge.svg)](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-production.yml)
```

Renders this:

# USDR: Grants One Stop Tool (GOST)


[![Code of Conduct](https://img.shields.io/badge/%E2%9D%A4-code%20of%20conduct-blue.svg?style=flat)](./CODE_OF_CONDUCT.md) [![License](https://img.shields.io/badge/license-Apache%202.0-blue.svg)](./LICENSE) [![Build Status](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-staging.yml/badge.svg)](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-staging.yml) [![Build Status](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-production.yml/badge.svg)](https://github.com/usdigitalresponse/usdr-gost/actions/workflows/deploy-production.yml)